dev/PushNotifications/PushNotificationBackgroundTask.cpp

```diff
@@ -0,0 +1,23 @@
+﻿#pragma once
+
+#include "pch.h"
+
+#include "PushNotificationBackgroundTask.h"
+
+#include <winrt/base.h>
+#include <winrt/windows.networking.pushnotifications.h>
+#include "PushNotificationReceivedEventArgs.h"
+#include "externs.h"
+
+using namespace winrt::Windows::ApplicationModel::Background;
+using namespace winrt::Windows::Networking::PushNotifications;
+
+wil::unique_handle g_waitHandleForArgs;
+winrt::Microsoft::Windows::PushNotifications::PushNotificationReceivedEventArgs g_activatedEventArgs{ nullptr };
```

Comment:
Nit: if you are already doing `using namespace`, why not doing the same for `winrt::Microsoft::Windows::PushNotifications`?